homeworks/07-mars-viewer/src/features/mars-viewer/marsSlice.ts

```diff
@@ -0,0 +1,81 @@
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../../app/store';
+import nasa, { NasaResponse } from '../../api/nasa';
+
+export type Status = 'idle' | 'loading';
```

Comment:
Just a small note if some values will be used in runtime like in this case I suggest use enum instead of union types, so if it's some status from backend will change there would be less changes in codebase since real values of statuses are located only in one place.